ray/README.md

```diff
@@ -1,68 +1,82 @@
+This method requires that you have [kustomize](https://kustomize.io/) and the [oc client](https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html) installed in your environment. Then, from within the `ray/` directory of this repo you can run the following:
 
 ```bash
-$ oc kustomize ray/operator/base > operator_deployment.yaml
+cd operator/base
+oc kustomize > deploy_kuberay.yaml
+oc create -f deploy_kuberay.yaml
 ```
-```bash
-$ oc apply -f operator_deployment.yaml -n <your-namespace>
+
+## Install via KfDef:
+
+Alternatively, you can include the KubeRay components in your ODH KfDef. This will ensure that all of the components above get installed as part of your ODH deployment.  
+
+```yaml
+ # KubeRay
+  - kustomizeConfig:
+      repoRef:
+        name: manifests-contrib
+        path: ray/operator
+    name: ray-operator
 ```
+You can find a minimal [KfDef here](ray-minimal-kfdef.yaml), that can be used to just install the core ODH components, along with everything you'll need to use KubeRay.  
+
+
+
+
+## Confirm the operator is running:
+
+Once installed, you can confirm the KubeRay operator has been deployed correctly with `oc get pods`.
 
-#### Confirm the operator is running 
 
 ```
-$ oc get pods 
+$ oc get pods
 NAME                               READY   STATUS    RESTARTS        AGE
-ray-operator-867bc855b7-2tzxs      1/1     Running   0               4d19h
+kuberay-operator-867bc855b7-2tzxs      1/1     Running   0               4d19h
 
 ```
 
-#### Create a ray cluster 
+## Create a test cluster 
 
+Now that the operator is running, let's create a small Ray cluster and make sure the operator can handle the request correctly. From whatever namespace you want to use you can run the following command:
 
 ```bash
-$ oc kustomize ray/cluster/base > cluster_deployment.yaml
-```
-```bash
-$ oc apply -f cluster_deployment.yaml -n <your-namespace>
+$ oc apply -f tests/resources/ray/ray-test-cluster-test.yaml
 ```
 
-#### Confirm the cluster is running 
+## Confirm the cluster is running 
 ```
-$ oc get pods 
-NAME                               READY   STATUS    RESTARTS        AGE
-ray-cluster-head-2f866             1/1     Running   0               36m
+$ oc get RayCluster 
+NAME                   DESIRED WORKERS   AVAILABLE WORKERS   STATUS   AGE
+kuberay-cluster-test   1                 2                   ready    13s
 
 ```
 
-Once the cluster is running you should be able to connect to it to use ray in a python script or jupyter notebook by using `ray.init(ray://<Ray_Cluster_Service_Name>:10001)`. 
+Once the cluster is running you should be able to connect to it to use ray in a python script or jupyter notebook by using `ray.init(ray://kuberay-cluster-test-head-svc:10001)`. 
 ```python 
 import ray
-ray.init(ray://<Ray_Cluster_Service_Name>:10001)
+ray.init('ray://kuberay-cluster-test-head-svc:10001')
```

Comment:
@MichaelClifford I don't think this should block merging of this PR, but I got this error in my notebook:
```
[RuntimeError: Client Ray installation incompatible with server: client is 2022-12-06, server is 2022-10-05](ray://kuberay-cluster-test-head-svc:10001'</span><span)
```

This was after doing a simple `!pip install ray` in the notebook using the out of the box standard data science ODH notebook image.

We may want to upgrade the default Ray cluster node images we use to be more up to date.

Reply:
Thanks @accorvin, 

Yeah the version of Ray in the notebook env has to match that in the worker nodes. I think I made the assumption here that the user would be using the "codeflare notebook" that has the matching ray version.

Reply:
Makes sense. We should document that assumption.

Reply:
For now I can add a note here that says which version of Ray to install